QA_Robot_Practice/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add *.xml and others extentions into .gitignore and comit it into the repo. Delete .idea folder from the repo